src/main/java/org/apache/commons/collections4/bidimap/DualLinkedHashBidiMap.java

```diff
@@ -26,9 +26,9 @@
 import org.apache.commons.collections4.BidiMap;
 
 /**
- * Implementation of {@code BidiMap} that uses two {@code LinkedHashMap} instances.
+ * Implementation of {@link BidiMap} that uses two {@link LinkedHashMap} instances.
```

Comment:
This might have been intentional? Some people prefer to avoid adding too many links in the Javadoc (forgot where I saw a presentation about that, but it was in some guide for writing Javadocs where the author mentioned to keep the number of links low, so users are not distracted).

Reply:
That's silly IMO, these links are fine.

Reply:
I found the document I mentioned earlier. It was not exactly a presentation, but the Oracle docs for javadocs: https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide (where it says "Use in-line links economically"). There's also a similar style guide in UI/UX, to avoid cluttering user interfaces. However, I'm fine whichever way :+1: